app/views/RoomView/index.js

```diff
@@ -490,15 +499,15 @@ export default class RoomView extends LoggedView {
 	renderActions = () => {
 		const { room } = this.state;
 		const {
-			user, showActions, showErrorActions, navigation
+			user, showErrorActions, navigation
 		} = this.props;
 		if (!navigation.isFocused()) {
 			return null;
 		}
 		return (
 			<React.Fragment>
-				{room._id && showActions
-					? <MessageActions room={room} user={user} />
+				{room._id
+					? <MessageActions room={room} user={user} setRef={ref => this.setRef(ref)} />
```

Comment:
Creating arrow functions on render is a bad practice and it should be avoided.
This creates functions on every render.

You should always be using the class function directly like this:
`setRef={this.setRef}`

Reply:
did bind `this.setRef` and then used like you suggested `setRef={this.setRef}`